calibration/CTDMOG/CGINS-CTDMOG-13430__20161015.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13430,CC_ptcb1,-2.743142e-004,
 37-13430,CC_ptcb2,0.00E+00,
 37-13430,CC_ptempa0,1.529769e+002,
-37-13430,CC_ptempa1,-6.717459e-002,
+37-13430,CC_ptempa1,-6.717458e-02,
 37-13430,CC_ptempa2,4.759066e-007,
-37-13430,CC_wbotc,1.5313e-007,
+37-13430,CC_wbotc,1.531297e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT